examples/features/grid.js

```diff
@@ -264,7 +280,21 @@ Ext.application({
             region: 'east',
             store: featStore2,
             columns: [
-                {xtype: 'gx_symbolizercolumn', width: 40},
+                {
+                    xtype: 'widgetcolumn',
+                    width: 40,
+                    widget: {
+                        xtype: 'gx_renderer'
+                    },
+                    onWidgetAttach: function(column, gxRenderer, record) {
+                        // update the symbolizer with the related feature
+                        var feature = record.olObject;
+                        gxRenderer.update({
+                            feature: feature,
+                            symbolizers: featRenderer.determineStyle(record)
+                        });
```

Comment:
Confirmed this is not an issue. It's taken from the geometry type of the feature correctly.